test/format.spec.js

```diff
@@ -78,6 +84,22 @@ describe('format', () => {
       expect(trace.resource).to.equal('resource')
     })
 
+    it('should discard user-set tags with name HOSTNAME_KEY by default', () => {
```

Comment:
`user set`

Reply:
I believe it should be `user-set` because it's a compound adjective.

I've now changed it to `user-defined` to make it clearer, but I don't really mind removing the hyphen.

Reply:
The more you know ⭐️